src/common/mpu6050.rs

```diff
@@ -30,6 +46,7 @@ pub(crate) fn register_models(registry: &mut ComponentRegistry) {
 
 const READING_START_REGISTER: u8 = 59;
 const STANDBY_MODE_REGISTER: u8 = 107;
+const MAX_U16: f64 = 32768.0;
```

Comment:
Is there a rust intrinsic for this? Sort of like C++ has `std::numeric_limits<T>::max`?